lib/balena.coffee

```diff
@@ -104,8 +99,22 @@ getSdk = (opts = {}) ->
 		sdkInstance: sdk
 	}
 
-	for moduleName, moduleFactory of sdkTemplate
-		sdk[moduleName] = moduleFactory(deps, opts)
+	[
+		'auth',
+		'models'
+		'logs'
+		'settings'
+	].forEach (moduleName) ->
+		Object.defineProperty(sdk, moduleName, {
+			enumerable: true,
+			configurable: true,
+			get: ->
+				moduleFactory = require("./#{moduleName}")
```

Comment:
It seems that browserify doesn't really like this way of imports.
How about doing
```
sdkTemplate =
	auth: -> require('./auth')
	models: -> require('./models')
	logs: -> require('./logs')
	settings: -> require('./settings'
```
and then in here just call
```
moduleFactory = sdkTemplate["./#{moduleName}"]()
```
?

Reply:
Ok, I was going to ask about browserify, I'll switch it over to that style - I was already considering using that style to remove the distinction between `.default` and non `.default` handling and if it keeps browserify happy then it's a no-brainer